dll/win32/ntdll_vista/threadpool.c

```diff
@@ -0,0 +1,2128 @@
+/*
```

Comment:
Can you mention this file in README.WINE please?

Reply:
done